homeassistant/components/sensor/vera.py

```diff
@@ -146,11 +146,12 @@ def state_attributes(self):
 
         if self.vera_device.is_trippable:
             last_tripped = self.vera_device.refresh_value('LastTrip')
-            trip_time_str = time.strftime(
-                "%Y-%m-%d %H:%M",
-                time.localtime(int(last_tripped))
-            )
-            attr[ATTR_LAST_TRIP_TIME] = trip_time_str
+            if last_tripped is not None:
+                utc_time = dt_util.utc_from_timestamp(int(last_tripped))
+                attr[ATTR_LAST_TRIP_TIME] = dt_util.datetime_to_local_str(
```

Comment:
What's the reason for using local time? You can't know in which timezone the browser will be.